src/interfaces/ProviderInterfaces.sol

```diff
@@ -83,3 +85,17 @@ interface ISwEth {
 
     function getRate() external view returns (uint256);
 }
+
```

Comment:
Just a comment for now but having these in one file may not be a good idea forever